### ISSUE TITLE
Issue mengenai Front End Form Edit Acara

### ISSUE THREAD

**Issue by MonkeyHiko** [SwingsNFI/Project_Absen_Telkom-2#17]
Form edit acara sudah harus memiliki **fitur pilih tanggal dan jam** yang mudah digunakan oleh user. Sedangkan yang ada saat ini hanya text field biasa. Dan seharusnya pilih **nama acaranya** berupa **drop down** yang list terakhir bertuliskan tambah acara lain. 

Link : https://github.com/SwingsNFI/Project_Absen_Telkom-2/blob/master/ProjectAbsenTelkom2/Frontend/Admin%20Page.rar

**Comment by SwingsNFI** [comment]
Yup memang seharusnya kayak begitu. Ohh yaa btw nanti selesain acaranya gimana?

**Comment by SwingsNFI** [comment]
@Azis202017 kamu tambahin lagi, kalau udah, aku close issue

**Comment by MonkeyHiko** [comment]
> Yup memang seharusnya kayak begitu. Ohh yaa btw nanti selesain acaranya gimana?

Acara tidak perlu diselesaikan ( **no need to be deleted** ) agar dapat dijadikan sebagai history.

**Comment by SwingsNFI** [comment]
nutup acaranya gimana?
